packages/financial-templates-lib/src/clients/OptimisticOracleClient.ts

```diff
@@ -144,33 +195,31 @@ export class OptimisticOracleClient {
       return hasDispute === undefined;
     });
 
-    const unsettledProposals = (
```

Comment:
This logic was actually uneccessary as we can simple filter out any events that overlap with `Settle` events.

Reply:
nice.